Makefile

```diff
@@ -122,12 +124,20 @@ build-itest-race:
 	@$(call print, "Building itest binary for ${backend} backend.")
 	CGO_ENABLED=0 $(GOTEST) -v ./itest -tags="$(DEV_TAGS) $(RPC_TAGS) integration $(backend)" -c -o itest/itest.test$(EXEC_SUFFIX)
 
-#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin
-install:
+#? install: Build and install lnd and lncli binaries, place them in $GOPATH/bin.
+install-binaries:
```

Comment:
nit: the help hint for this make goal is now outdated. Also, looking at all other help hints, we don't use full stops at the end of sentences everywhere else, so let's make this and the ones below consistent with the rest of the file.